app/services/session_encryptor_error_handler.rb

```diff
@@ -0,0 +1,5 @@
+class SessionEncryptorErrorHandler
```

Comment:
This could also be an anonymous class or a class defined in the initializer.

Reply:
If we don't expect to call this from anywhere other than the session store initializer, maybe it makes sense to define it there? I don't have strong feelings about this.